scripts/index.js

```diff
@@ -51,6 +51,17 @@ function coldTimeInTheCity(cities) {
 coldTimeInTheCity(cities, "temperature");
 
 
+//**************cities #2 *****************
+
+function whosTownIsThis(cities) {
+    let notMyTown = citites.filter(val => {
```

Comment:
Off to a great start. Right now cities on line 56 and citites on line 57 aren't spelled the same. After that you were well on your way to working your way through this problem. Here's an example...
`let cities = ['cities', 'counties', 'countries'];
function whosTownIsThis(cities) {
    let notMyTown = cities.filter(val => {
        return val === 'cities';
    })
    return notMyTown;
}
whosTownIsThis(cities);`